packages/design-system/src/components/Dropdown/Dropdown.spec.tsx

```diff
@@ -20,6 +20,14 @@ context('<Dropdown />', () => {
 		cy.getByTest('dropdown.menu').should('not.be.visible');
 	});
 
+	it('should hide the menu clicking on a button', () => {
+		cy.mount(<WithIcons />);
+		cy.getByTest('dropdown.button').click();
+		cy.getByTest('dropdown.menu').should('be.visible');
+		cy.get('button[data-test="dropdown.menuitem"]').click();
```

Comment:
Isn't enough?
```suggestion
		cy.getByTest("dropdown.menuitem").click();
```

Reply:
The first item is a link, not the button

Reply:
Could be nice to test both :)

Reply:
If we click on the link, we are redirected and the test fall into space, do you have a trick to prevent the navigation or something ? (without altering the story)

Reply:
🤷🏻 
```js
cy.getByTest("dropdown.menuitem").eq(0).invoke('removeAttr', 'target').click();
```

Reply:
Thanks for the trick 👍🏼 ❤️